packages/main/src/Dialog.js

```diff
@@ -20,6 +21,16 @@ import dialogCSS from "./generated/themes/Dialog.css.js";
  */
 const STEP_SIZE = 16;
 
+/**
+ * Defines the icons corresponding to the dialog's state.
+ */
+const ICON_PER_STATE = {
+	Error: "error",
+	Warning: "alert",
+	Success: "sys-enter-2",
+	Information: "information",
```

Comment:
Use the enum ValueState's values as keys to this object like this:

```js
const ICON_PER_STATE = {
	[ValueState.Error]: "error",
	[ValueState.Warning]: "alert",
	[ValueState.Success]: "sys-enter-2",
	[ValueState.Information]: "information",
};
```

This code does the same thing as you have written, but it more clearly shows how each icon maps to a particular value of the enum ValueState.